docker/Dockerfile.ort-default-stable-torch190-onnxruntime181-cu102-cudnn7-devel-ubuntu18.04

```
@@ -0,0 +1,34 @@
+# ONNX Runtime Training Module for PyTorch
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# CUDA development image for building sources
+FROM nvidia/cuda:10.2-cudnn7-devel-ubuntu18.04 as builder
+
+ARG TORCH_CUDA_VERSION=cu102
+ARG ONNXRUNTIME_TRAINING_VERSION=1.8.1
+ARG TORCH_VERSION=1.9.0
+ARG TORCHVISION_VERSION=0.10.0
+
+# Install and update tools to minimize security vulnerabilities
+RUN apt-get update
+RUN apt-get install -y software-properties-common wget apt-utils patchelf git libprotobuf-dev protobuf-compiler cmake
+RUN unattended-upgrade
+RUN apt-get autoremove -y
+
+# Python and pip
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 1
+RUN apt-get install -y python3-pip
+RUN update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 1
+RUN pip install --upgrade pip
+
+# PyTorch
+RUN pip install onnx ninja
+RUN pip install torch==${TORCH_VERSION}+${TORCH_CUDA_VERSION} torchvision==${TORCHVISION_VERSION}+${TORCH_CUDA_VERSION} -f https://download.pytorch.org/whl/torch_stable.html
+
+# ORT Module
+RUN pip install onnxruntime-training==${ONNXRUNTIME_TRAINING_VERSION}
+
+RUN pip install torch-ort
+
```

```suggestion
RUN python -m torch_ort.configure
```

Should we call the PyTorch CPP extension build?